OneCommunityGlobal/HighestGoodNetworkApp
# Gayathri_lint_fix_for_common_component

# Description
Lint Fix for Common Component src/components/common/**

## Related PRS (if any):
Removed the component from  .eslintignore](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1480)
…

## Main changes explained:
Executed the link checker
Fixed changes manually apart from fix done by lint--fix
Removed the component from  .eslintignore

## How to test:
check into current branch for Front End
checkout any branch for Backend
Run npm install and npx eslint src/components/common/* to make sure there is no lint errors 
Run npm run test to make sure that all tests pass
Run npm run start:local
log as any user
Verify that common components such as checkbox, form, dropdown, etc work properly

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/163784958/f46bbf86-8ca6-432d-8210-91b36a3d718b)

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/163784958/e8402452-0379-477f-86fb-c4621f27120a)


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/163784958/40845881-aaae-404b-a60d-892bd5065bd8





## Note:
Include the information the reviewers need to know.
if you have an Eslint extension in your Vs code you do not need to give npx eslint src/components/common/ (command)
